pipeline-templates/prepare-integration-test-steps.yml

```diff
@@ -1,13 +1,8 @@
 steps:
   - task: UseDotNet@2
-    displayName: 'Install dotnet core 3.0.x because macos is not up to date'
+    displayName: 'Install dotnet core 3.1.x to make dotnet tool path available in path'
```

Comment:
This is possibly no longer needed, this was only still needed because the macosx agent was not up to date